cabal-install/src/Distribution/Client/CmdRepl.hs

```diff
@@ -338,7 +338,7 @@ addDepsToProjectTarget deps pkgId ctx =
       | packageId pkg /= pkgId = SpecificSourcePackage pkg
       | SourcePackage{..} <- pkg =
         SpecificSourcePackage $ pkg { srcpkgDescription =
-          srcpkgDescription & (\f -> L.allCondTrees $ traverseCondTreeC f)
+          srcpkgDescription &  (L.traverseBuildInfos . L.targetBuildDepends)
```

Comment:
And/or a couple of lines of code comments here? Whichever you prefer. The explanation in the commit would be historical "previously there was L.allCondTrees $ traverseCondTreeC, but it was bad and now it's good", while the comment here would be ahistorical: "targetBuildDepends makes sense here because reasons, and traverseBuildInfos then completes the wondrous transformation, yielding this and that".

Reply:
Done. Do you think it explains adequately?

Reply:
🥇